pkgs/games/humblebundle/antichamber.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv, stdenv_32bit, lib, pkgsi686Linux, fetchHumbleBundle, unzip}:
+
+stdenv_32bit.mkDerivation rec {
+  name = "antichamber-1.1";
+
+  src = fetchHumbleBundle {
+    name = "antichamber_1.01_linux_1392664980.sh";
+    machineName = "antichamber_linux";
+    md5 = "37bca01c411d813c8729259b7db2dba0";
+  };
+
+  dontStrip = true;
+  phases = ["installPhase"];
+
+  installPhase = let
+    rpath = with pkgsi686Linux; lib.makeLibraryPath [
+      "$dest/Binaries/Linux"
+      xorg.libX11
+      xorg.libXi
+      stdenv.cc.cc.lib
+      libpulseaudio
+      libvorbis
+      libogg
+      mesa
+    ];
+   in ''
+    set -x
+    dest="$out/share/antichamber"
+
+    # Unpack binaries and data into $dest
+    mkdir -p "$dest"
+    ${unzip}/bin/unzip $src "data/*" -d $dest && [ $? -le 2 ]
+    mv $dest/data/*/* $dest
```

Comment:
I'd assume here that `data/*` is just a single directory, right? If yes, see the above comment about `unzip`, which in this case would make even more sense to pass that subdirectory to `unzip`.